CMakeLists.txt

```diff
@@ -1,30 +1,91 @@
 cmake_minimum_required(VERSION 2.8.3)
+
 project(image_undistort)
 
-find_package(catkin_simple REQUIRED)
```

Comment:
I followed the realsense driver as an example when setting up building nodelets which didn't use catkin simple and didn't switch back after I got things running